docs/en/latest/how-to-build.md

```diff
@@ -21,351 +25,218 @@ title: Installing Apache APISIX
 #
 -->
 
-This guide walks you through how you can build and get Apache APISIX running on your environment. Please refer the [Getting Started](./getting-started.md) guide for a quick walkthrough on running Apache APISIX.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Step 1: Install Apache APISIX
+This guide walks you through how you can install and run Apache APISIX in your environment.
 
-Apache APISIX can be installed via the [RPM package](#installation-via-rpm-repository-centos), [Docker image](#installation-via-docker), [Helm Chart](#installation-via-helm-chart) or the [source release package](#installation-via-source-release-package). You can install via any one of these options.
+Refer to the [Getting Started](./getting-started.md) guide for a quick walk-through on running Apache APISIX.
 
-### Installation via RPM Repository (CentOS)
+## Installing etcd
 
-> This installation method is suitable for CentOS 7 and CentOS 8.
+APISIX uses [etcd](https://github.com/etcd-io/etcd) to save and synchronize configuration. Before installing APISIX, you need to install etcd on your machine.
 
-If the official OpenResty repository is **not installed yet**, the following command will help you automatically install both OpenResty and Apache APISIX repositories.
+It would be installed automatically if you choose the Docker or Helm install method while installing APISIX. If you choose a different method or you need to install it manually, follow the steps shown below:
```

Comment:
We don't need to install etcd when installing apisix by docker-compose and Helm.
So So this part is unnecessary for many users

Reply:
We have mentioned that it would be installed for Docker and Helm. Where should we mention etcd installation instead? Just so that users installing via RPM also have it. I was suggested to add etcd installation first [`9cde814` (#6964)](https://github.com/apache/apisix/pull/6964/commits/9cde814e87227de42e0c011306f14f2151b46206#r861433102)